tests/test_Manifest.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import json
+import os
+
+import pytest
+
+from rsconnect.bundle import Manifest
+from rsconnect.exception import RSConnectException
+
+cur_dir = os.path.dirname(__file__)
+html_manifest_json_file = os.path.join(cur_dir, "testdata", "Manifest", "html_manifest.json")
+
+
+@pytest.mark.skipif(sys.platform in ("win32", "win64"), reason="Backslash vs forward slash")
```

Comment:
Skipping these tests on Windows is OK if you create an alternate version of the tests that runs only on Windows.